CODEOWNERS

```diff
@@ -283,7 +283,7 @@ homeassistant/components/monoprice/* @etsinko @OnFreund
 homeassistant/components/moon/* @fabaff
 homeassistant/components/motion_blinds/* @starkillerOG
 homeassistant/components/mpd/* @fabaff
-homeassistant/components/mqtt/* @home-assistant/core @emontnemery
+homeassistant/components/mqtt/* @home-assistant/core @emontnemery @qistoph
```

Comment:
The implication here is that you want to be notified each time an issue is reported on the MQTT integration and help out to fix bugs and review PRs, is that your intention?

Whichever the case, open a separate PR for that.

Reply:
Thanks for the feedback. That was indeed not my intention, but done because of an error suggesting I had to in the `pre-commit` hook. I'll remove myself from codeowners in a PR update.